package.json

```diff
@@ -66,7 +66,7 @@
   "dependencies": {
     "@dashevo/dashcore-lib": "~0.18.12",
     "@dashevo/dashd-rpc": "^2.0.0",
-    "@dashevo/dpp": "~0.15.0",
+    "@dashevo/dpp": "~0.16.0-dev",
```

Comment:
You can use just a "0.16-dev" channel. It's like "latest" but for 0.16 dev versions.